_includes/assets/sass/_typography.scss

```diff
@@ -116,4 +120,20 @@ h4, .h4-style {
 	line-height: govuk-px-to-rem(16px);
 	text-align: center;
 	color: $red;
+}
+
+.large-body-text {
+	font-size: $lg-body-font-size-mobile;
+	line-height: $lg-body-line-height-mobile;
+	font-weight: $base-font-weight-regular;
+
+	@media only screen and (min-width : $tablet) {
+		font-size: $lg-body-font-size;
+		line-height: $lg-body-line-height;
+	}
+}
+
+// Colours
+.white-text {
+	color: $white;
 }
```

Comment:
Blank line at the end of files is part of the Drupal coding standards. I don't really care as long as it works, but it does get rid of these annoying messages.

Reply:
Great, totally makes sense. Added.